lambda-functions/jira-bot-filter-task-runner/condition-checker.js

```diff
@@ -0,0 +1,51 @@
+const moment = require('moment');
+const config = require('./config');
+
+lastHumanActivityDate = issue => {
+  const humanHistories = issue.changelog.histories.filter(history => {
```

Comment:
i would suggest the variable name to be `nonAuthorHistories`. an author is still human! hehehe